modules/reference/pages/properties.adoc

```diff
@@ -0,0 +1,6361 @@
+
+Address and port of admin server
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/model::broker_endpoint
+
+*Default:* '{model::broker_endpoint(net::unresolved_address("127.0.0.1", 9644))}'
+
+=== admin_api_doc_dir
+
+Admin API doc directory
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* '/usr/share/redpanda/admin-api-doc'
+
+=== admin_api_tls
+
+TLS configuration for admin HTTP server
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/endpoint_tls_config
+
+*Default:* None
+
+=== cloud_storage_cache_directory
+
+Directory for archival cache. Should be present when `cloud_storage_enabled` is present
+
+*Requires Restart:* Yes
+
+*Nullable:* Yes
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* None
+
+=== crash_loop_limit
+
+Maximum consecutive crashes (unclean shutdowns) allowed after which operator intervention is needed to startup the broker. Limit is not enforced in developer mode.
+
+*Requires Restart:* Yes
+
+*Nullable:* Yes
+
+*Visibility:* user
+
+*Type:* integer
+
+*Accepted values:* [0, 4294967295]
+
+*Default:* 5
+
+=== data_directory
+
+Place where redpanda will keep the data
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* string
+
+*Default:* None
+
+=== developer_mode
+
+Skips most of the checks performed at startup, not recomended for production use
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* tunable
+
+*Type:* boolean
+
+*Default:* False
+
+=== emergency_disable_data_transforms
+
+Override the cluster enablement setting and disable WebAssembly powered data transforms. Only used as an emergency shutoff button.
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* boolean
+
+*Default:* False
+
+=== empty_seed_starts_cluster
+
+If true, an empty seed_servers list will denote that this node should form a cluster. At most one node in the cluster should be configured configured with an empty seed_servers list. If no such configured node exists, or if configured to false, all nodes denoted by the seed_servers list must be identical among those nodes' configurations, and those nodes will form the initial cluster.
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* boolean
+
+*Default:* True
+
+=== kafka_api
+
+Address and port of an interface to listen for Kafka API requests
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/config::broker_authn_endpoint
+
+*Default:* {'address': 'net::unresolved_address("127.0.0.1", 9092)', 'authn_method': 'std::nullopt'}
+
+=== kafka_api_tls
+
+TLS configuration for Kafka API endpoint
+
+*Requires Restart:* Yes
+
+*Nullable:* No
+
+*Visibility:* user
+
+*Type:* #/definitions/endpoint_tls_config
+
+*Default:* None
+
+=== memory_allocation_warning_threshold
+
+Enables log messages for allocations greater than the given size.
```

Comment:
**ENG:** Would it be more accurate to say: 
Specifies the threshold for memory allocations and enables log messages when this threshold is exceeded.